src/design-system-team.md.njk

```diff
@@ -15,10 +15,11 @@ If you want to contact the team you can
 ## Members
 
 - Chris Thomas – Senior Interaction Designer
+- Charlotte Downs – Interaction Designer
```

Comment:
The list orders our first names alphabetically so yours should be top of the list :)